sandy/decay.py

```diff
@@ -159,6 +159,14 @@ def get_half_life(self, with_uncertainty=True):
         922350   2.22102e+16
         942400   2.07108e+11
         Name: HL, dtype: float64
+        
+        Stable nuclide:
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", 260560)
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> rdd.get_half_life(False)
+        ZAM
+        260560   0.00000e+00
```

Comment:
Please, add a note (check or ask how to write notes) in the docstring that a zero will be given for the half-life of stable nuclides. This is in agreement with the ENDF6 manual